.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+Thank you for contributing to JanusGraph.
+
+In order to streamline the review of the contribution we ask you
+to ensure the following steps have been taken:
+
+### For all changes:
+- [ ] Is there an issue associated with this PR? Is it referenced 
+     in the commit message?
+
+- [ ] Does your PR title ends with #xyz where xyz is the issue number you are trying to resolve?
```

Comment:
I'm talking about this:

> Does your PR title ends with #xyz where xyz is the issue number you are trying to resolve?

I am saying that putting the issue # at the end of a PR title is confusing, because of the reasons above: it's not auto-linked, and GitHub appends the PR # to the end, so there are two numbers in the title: the issue it's addressing, and the PR itself.

I am suggesting that instead of putting the issue # in the PR title, we put it in the PR description instead, which addresses all of the above issues. Having it in the commit also makes sense, but that's not what I'm referring to.